pymc_experimental/distributions/continuous.py

```diff
@@ -0,0 +1,175 @@
+from typing import List, Optional, Tuple, Union
+
+import numpy as np
+import aesara
+import aesara.tensor as at
+from pymc.aesaraf import floatX
+from aesara.tensor.var import TensorConstant, TensorVariable
+from aesara.tensor.random.op import RandomVariable
+
+from aesara.tensor.random.basic import gengamma
```

Comment:
I guess the gengamma is not available yet in the Aesara version pinned in PyMC?

Reply:
Its been 4 days but I just tried setting up a new clean virtual environment and it seems like its there!

Reply:
all set with the latest commit!